src/main/kotlin/com/openlattice/chronicle/notifications/NotificationStatus.kt

```diff
@@ -0,0 +1,16 @@
+package com.openlattice.chronicle.notifications
+
+/**
+ *  @author Todd Bergman <todd@openlattice.com>
+ *  https://support.twilio.com/hc/en-us/articles/223134347-What-are-the-Possible-SMS-and-MMS-Message-Statuses-and-What-do-They-Mean-
+ *  Finalized Message Delivery Status
+ *  After a message has been sent, the following final delivery status options are possible:
+ */
+
+enum class NotificationStatus {
+    delivered,
```

Comment:
Should be caps.

Reply:
or PascalCase

Reply:
This enum is meant to represent Twilio constants:

https://support.twilio.com/hc/en-us/articles/223134347-What-are-the-Possible-SMS-and-MMS-Message-Statuses-and-What-do-They-Mean-

Reply:
Got it. Makes sense.